src/SortingAlgorithms.jl

```diff
@@ -571,4 +595,18 @@ function sort!(v::AbstractVector, lo::Int, hi::Int, ::TimSortAlg, o::Ordering)
     return v
 end
 
+function sort!(v::AbstractVector, lo::Int, hi::Int, ::CombSortAlg, o::Ordering)
+    interval = (3 * (hi-lo+1)) >> 2
+
+    while interval > 1
+        @inbounds for j in lo:hi-interval
+            a, b = v[j], v[j+interval]
+            v[j], v[j+interval] = lt(o, a, b) ? (a, b) : (b, a)
```

Comment:
This will swap if `a == b`. While this is not technically a bug because the algorithm is already advertised as unstable, it would be better to swap only if needed.
```suggestion
            a, b = v[j], v[j+interval]
            v[j], v[j+interval] = lt(o, b, a) ? (b, a) : (a, b)
```

Reply:
Good one!